index.html

```diff
@@ -0,0 +1,165 @@
+        <!-- dog 3 -->
+        <div class="col-1">
+          <img src="./images/jack-card.jpg" onclick="clickPhoto('Jack','beagle','$123.45')" alt="beagle">
+          <h3>Jack</h3>
+          <p><b>cost to adopt</b> $123.45</p>
+          <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum,qui doluptae</p>
+          <a class="button" onclick="dogFees(123.45)" >Adopt</a>
+        </div>
+        <!-- dog 4 -->
+        <div class="col-1">
+          <img src="./images/duffy-card.jpg" onclick="clickPhoto('Duffy','pitbull', '$123.45')" alt="dog with white face">
+          <h3>Duffy</h3>
+          <p><b>cost to adopt:</b> $123.45</p>
+          <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum,qui doluptae</p>
+          <a class="button" onclick="dogFees(123.45)" >Adopt</a>
+        </div>
+        <!-- dog 5 -->
+        <div class="col-1">
+          <img src="./images/lucas-card.jpg" onclick="clickPhoto('Lucas','shepherd', '$123.45')" alt="black dog with upright ears">
+          <h3>Lucas</h3>
+          <p><b>cost to adopt:</b> $123.45</p>
+          <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum,qui doluptae</p>
+          <a class="button" onclick="dogFees(123.45)" >Adopt</a>
+        </div>
+        <!-- dog 6 -->
+        <div class="col-1">
+          <img src="./images/jake-card.jpg" onclick="clickPhoto('Jake','Lab','$123.45')" alt="adult lab">
+          <h3>Jake</h3>
+          <p><b>cost to adopt:</b> $123.45</p>
+          <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum,qui doluptae</p>
+          <a class="button" onclick="dogFees(123.45)" >Adopt</a>
+        </div>
+        <!-- dog 7
+        <div class="col-1">
+          <img src="./images/angus-card.jpg" onclick="clickPhoto('Angus','Boxer', '$123.45')" alt="brown dog withlarge head">
+          <h3>Angus</h3>
+          <p><b>cost to adopt:</b> $123.45</p>
+          <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum,qui doluptae</p>
+          <a class="button" onclick="dogFees(123.45)" >Adopt</a>
+        </div>-->
+        <!-- dog 8
+        <div class="col-1">
+          <img src="./images/violet-card.jpg" onclick="clickPhoto('Violet','Lab', '$123.45')" alt="brown lab puppy">
+          <h3>Violet</h3>
+          <p><b>cost to adopt:</b> $123.45</p>
+          <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum,qui doluptae</p>
+          <a class="button" onclick="dogFees(123.45)" >Adopt</a>
+          </div> -->
+      </div>
+    </section>
+    <div class="graybutton">
+      <a href="./dogs.html" style = "background-color: lightGray; color: darkblue; margin-top: 1rem; margin-left: 30rem;" target= "_blank"> See More Dogs</a><br>
```

Comment:
Try not to use inline styles, they make debugging and your styles difficult. 30 rem is far to high a number and is affecting other elements at smaller mobile resolutions.